R/io.R

```diff
@@ -0,0 +1,27 @@
+
+read_cohort <- function(cohortfile='data/pg_sankey_data_1_3_yag_dummy.csv'){
+  cohort <- read.csv(cohortfile)
+  cohort <- subset(cohort, select = -X)
+  cohort$SECTIONNAME.x <- StrCap(tolower(cohort$SECTIONNAME.x))
+  cohort$SECTIONNAME.y <- StrCap(tolower(cohort$SECTIONNAME.y))
+  return(cohort)
+}
+
+read_tables_data <- function(file){
+  tables_data <- fread(file) %>% select(-V1)
+  names(tables_data) <- c("X", "YAG", "subject_name", "SECTIONNAME", "sex", 
+                          "ethnicity", "current_region", "FSM", "prior_attainment",
+                          "count", "earnings_median", "threshold", "qualification_TR", 
+                          "group_name")
+  tables_data$SECTIONNAME[tables_data$group_name == "Radio broadcasting"] <- "INFORMATION AND COMMUNICATION"
+  tables_data$SECTIONNAME[tables_data$group_name == "Reproduction of recorded media"] <- "MANUFACTURING"
+  tables_data$SECTIONNAME <- StrCap(tolower(tables_data$SECTIONNAME))
+  return(tables_data)
+}
+
+
+cohort1 <- read_cohort('data/pg_sankey_data_1_3_yag_dummy.csv')
```

Comment:
I think these calls at the bottom should stay in the global file, leaving this script just with the functions?

Reply:
Fair enough, that just hits the source ordering though. By default R loads global.R and then the scripts in R/ I think? So would then just need to add a line in global.R sourcing the read_data.R script before reading in any data. But maybe this will shift around once we're getting everything the data into a reactive state anyway.